python/tests/integration/scripts/test_mass_nifti_pic.py

```diff
@@ -0,0 +1,264 @@
+    assert process.stderr == ""
+
+
+def test_missing_largest_id_arg():
+    """
+    Test running the script without the --largest_id argument.
+    """
+
+    process = run_integration_script([
+        'mass_nifti_pic.py',
+        '--profile', 'database_config.py',
+        '--smallest_id', '2',
+    ])
+
+    # Check return code, STDOUT and STDERR
+    message = 'ERROR: you must specify a largest FileID on which to run the' \
+              ' mass_nifti_pic.py script using -l or --largest_id option'
+    assert process.returncode == MISSING_ARG
+    assert message in process.stdout
+    assert process.stderr == ""
+
+
+def test_smallest_id_bigger_than_largest_id():
+    """
+    Test running the script with a --smallest_id higher than the --largest_id.
+    """
+
+    process = run_integration_script([
+        'mass_nifti_pic.py',
+        '--profile', 'database_config.py',
+        '--smallest_id', '6',
+        '--largest_id', '2'
+    ])
+
+    # Check return code, STDOUT and STDERR
+    message = 'ERROR: the value for --smallest_id option is bigger than value for --largest_id option'
+    assert process.returncode == INVALID_ARG
+    assert message in process.stdout
+    assert process.stderr == ""
+
+
+def test_on_invalid_file_id():
+    """
+    Test running the script on an invalid file ID.
+    """
+
+    process = run_integration_script([
+        'mass_nifti_pic.py',
+        '--profile', 'database_config.py',
+        '--smallest_id', '999',
+        '--largest_id', '999'
+    ])
+
+    # Check return code, STDOUT and STDERR
+    message = 'WARNING: no file in the database with FileID = 999'
+    assert process.returncode == SUCCESS
+    assert message in process.stdout
+    assert process.stderr == ""
+
+
+def test_on_file_id_that_already_has_a_pic():
+    """
+    Test running the script on a file that already has a pic.
+    """
+
+    process = run_integration_script([
+        'mass_nifti_pic.py',
+        '--profile', 'database_config.py',
+        '--smallest_id', '2',
+        '--largest_id', '2'
+    ])
+
+    # Check return code, STDOUT and STDERR
+    message = 'WARNING: there is already a pic for FileID 2. Use -f or --force to overwrite it'
+    assert process.returncode == SUCCESS
+    assert message in process.stdout
+    assert process.stderr == ""
+
+
+def test_force_option():
+    """
+    Test running the script on a file that already has the pic with option --force.
+    """
+
+    # database connection
+    db = get_integration_database_session()
+    file_pic_data = try_get_parameter_value_with_file_id_parameter_name(db, 2, 'check_pic_filename')
+    if file_pic_data:
+        # remove file from the file system before recreating it
+        # otherwise get Operation Not Permitted (specific to the test environment)
+        pic_to_remove = os.path.join('/data/loris/pic/', str(file_pic_data.value))
+        if os.path.exists(pic_to_remove):
+            os.remove(pic_to_remove)
```

Comment:
@maximemulder when I run the script with the force option without removing the pic file from the filesystem, I get a "Operation Not Permitted" so I have to first manually clean up the file from the testing environment and then run the script with `--force` option.

When I run the command directly on my test VM, the file can be overwritten with no issue. So that seems like this is an issue specific to our test environments. Not sure why though... 🤔